### ISSUE TITLE
Report source location when errors happen

### ISSUE THREAD

**Issue by andreypopp** [reworkcss/rework-inherit#7]
Currently it just throws an error with description, it would be nice if it also provides information from `node.position` to a user (`source` if available and `line`).

**Comment by jonathanong** [comment]
i haven't looked into rework's source stuff yet. would take a PR!

**Comment by jonathanong** [comment]
oh that's so simple. did you just want `err.position = position`? or did you want `err.start = position.start`, etc. there should be a de factor standard for this.

**Comment by tj** [comment]
yeah it would be nice if we could just try/catch and make err.message a snippet of near the failure, usually it's something obvious that you can see right away

**Comment by jonathanong** [comment]
the try/catch should go into rework, not the plugins, right? so you're okay with `err.position = position`?
